File: python/1133.py

x = int(input())
y = int(input())

if x > y :
  while (y+1) < x:
    y = y+1
    if y%5 == 2 or y%5 == 3 :
      print(y)
    
else:
  while (x+1) < y:
    x = x+1  
    if x%5 == 2 or x%5 == 3:
      print(x) 
                   

File: javascript/1133.js
// var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');

var lines =[10 ,18];
x = parseInt(lines.shift())
y = parseInt(lines.shift())

if(x>y){
    while(y<x){
        if(y%5==2 || y%5 ==3){
            console.log(y);
        }   
        y++;
    }
}else{
    while(x<y){
        if(x%5==2 || x%5 ==3){
            console.log(x);  
           
            }
    x++;            
        }
}




